src/object/schema_manager.c

```diff
@@ -5448,15 +5460,10 @@ sm_find_synonym (const char *name)
   AU_DISABLE (save);
   synonym_obj = obj_find_unique (synonym_class_obj, "unique_name", &value, AU_FETCH_READ);
   AU_ENABLE (save);
-  if (synonym_obj == NULL)
+
```

Comment:
When obj_find_unique() returns null, not all cases come with er_set() performed.
Therefore, as before, it is necessary to check whether synonym_obj is null.

Reply:
When synonym_obj is null, it's okay to not set an error.